src/Polytopes/Polyhedron/properties.jl

```diff
@@ -89,21 +99,17 @@ function faces(as::Type{T}, P::Polyhedron, face_dim::Int) where {T}
     if as == Polyhedron || as == Polyhedra
         PolyhedronFacePolyhedronIterator(P,face_dim-size(lineality_space(P),1))
     else
-        throw(ArgumentError("Unsupported `as` argument :" * string(as)))
+        throw(ArgumentError("Unsupported `as` argument:" * string(as)))
```

Comment:
In the future, I'd suggest to use string interpolation in this error messages (but this can be in another PR)
```suggestion
        throw(ArgumentError("Unsupported `as` argument: $(as)"))
```

Reply:
By the way, I have no idea how these threads of discussions are meant to converge.